.circleci/config.yml

```diff
@@ -7,14 +7,14 @@ orbs:
 default_matrix: &default_matrix
   matrix:
     parameters:
-      php-version: [ "8.0", "8.1", "8.2" ]
+      php-version: [ "8.0", "8.1", "8.2", "8.3", "8.4" ]
```

Comment:
Do we want to keep 8.0 and 8.1? They're no longer supported.

Reply:
8.1 still has security support but happy to remove 8.0